beta-src/src/utils/state/gameApiSlice/reducers/processUnitClick.ts

```diff
@@ -135,6 +135,7 @@ export default function processUnitClick(state, clickData) {
   } else if (inProgress) {
     if (unitID === clickData.payload.unitID) {
       resetOrder(state);
+      highlightMapTerritoriesBasedOnStatuses(state);
       if (type === "disband" || type === "retreat") {
         highlightMapTerritoriesBasedOnStatuses(state);
```

Comment:
Do we need the check on 139 after this addition?

Reply:
Removed it since I feel we would wan to reset territory highlights the 2nd time the unit is clicked.